CMakeLists.txt

```diff
@@ -323,6 +323,7 @@ if (ARB_WITH_PYTHON)
 endif()
 
 CPMAddPackage(NAME pugixml
+              FORCE TRUE
```

Comment:
Why is that needed? Could pugi be provided by the OS/base layer?

Reply:
I reintroduced the system package, if available. Changed the install method for pyarb to use rpaths, however, for it to find pugixml.so when linked with a spack installed version, for example.